src/CMakeLists.txt

```diff
@@ -130,10 +103,6 @@ set (NN_SOURCES
     utils/sleep.c
     utils/thread.h
     utils/thread.c
-    utils/thread_posix.h
```

Comment:
For clarity; this is working. I'm not sure exactly what you're saying here.

I'm saying that as it's written here, it's a stepping stone from where we are to where we want to be, and it currently works with both CMake and the existing autotools. Once completely switching over to CMake, we can then begin to rearchitect source code and CMake together, without having to worry how it affects the now-gone autotools.